roles/ansible-test-splitter/files/list_changed_targets.py

```diff
@@ -412,14 +429,60 @@ def build_result_struct(self, batches):
         return result
 
 
+def read_pullrequest_body(project_name, pull_request):
+    change_url = "https://api.github.com/repos/%s/pulls/%d" % (
+        project_name,
+        pull_request,
+    )
+    return [x for x in requests.get(change_url).json().get("body").split("\n") if x]
+
+
+ZUUL_EXTRA_TARGETS = "zuul-extra-targets"
+ZUUL_TARGETS = "zuul-targets"
+ZUUL_RELEASES = "zuul-releases"
+
+
+def read_pullrequest_zuul_override(project_name, pull_request):
+
+    desc = read_pullrequest_body(project_name, pull_request)
+
+    def _extract_data(line):
+        data = (":".join(line.split(":")[1:])).replace(",", " ")
+        return [x for x in data.split() if x]
+
+    result = {}
+    for line in desc:
+        for key in (ZUUL_EXTRA_TARGETS, ZUUL_TARGETS, ZUUL_RELEASES):
+            if line.lower().startswith(key + ":"):
+                result[key] = _extract_data(line)
+
+    return result
+
+
 if __name__ == "__main__":
     args = parse_args(sys.argv[1:])
 
     collections = [Collection(i) for i in args.collection_to_tests]
     collections_names = [c.collection_name() for c in collections]
 
+    ansible_releases = args.ansible_releases
+    zuul_targets, zuul_extra_targets = [], []
+    pr_request = read_pullrequest_zuul_override(args.project_name, args.pull_request)
+
+    if pr_request:
+        release_to_test = pr_request.get(ZUUL_RELEASES)
```

Comment:
I believe the `zuul-releases` is not used anymore. Can we remove it?

Reply:
It is